apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsAdjustment.java

```diff
@@ -0,0 +1,76 @@
+package gov.cms.bfd.model.rda;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+
+/** JPA class for the McsAdjustments table */
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@FieldNameConstants
+@IdClass(PreAdjMcsAdjustment.PK.class)
+@Table(name = "`McsAdjustments`", schema = "`pre_adj`")
+public class PreAdjMcsAdjustment {
+  @Id
+  @Column(name = "`idrClmHdIcn`", length = 15, nullable = false)
+  @EqualsAndHashCode.Include
+  private String idrClmHdIcn;
+
+  @Id
+  @Column(name = "`priority`", nullable = false)
+  @EqualsAndHashCode.Include
+  private short priority;
+
+  @Column(name = "`lastUpdated`")
+  private Instant lastUpdated;
+
+  @Column(name = "`idrAdjDate`")
```

Comment:
🍰  Could we get those handy comments about what these fields names mean in this class too?

Reply:
I grabbed those comments from the RDA API proto files.  Not all of the fields have comments though.